single-sign-on/index.md

```diff
@@ -36,9 +36,15 @@ In addition, all email addresses should be added to your IdP.
 
 To configure SSO, log into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} to obtain the **ACS URL** and **Entity IDs** to complete the IdP server configuration process. You can only configure SSO with a single IdP.  When this is complete, log back into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} and complete the SSO enablement process.
 
-> **Note:**
+> **Important**
 >
-> IdP initiated connections are not supported at this time
+> If your IdP setup requires an Entity ID and the ACS URL, you must select the
+> SAML tab in the Authentication Method section. For example, if your Azure AD
+> setup uses SAML configuration within Azure AD, you must select SAML. If you
+> are [configuring Open ID Connect with Azure AD](https://docs.microsoft.com/en-us/powerapps/maker/portals/configure/configure-openid-settings){:
```

Comment:
We don't need to the CTA ID for the URL as we are not redirecting users to a Docker website

Reply:
Thanks Usha! Removing it